pkg/apis/monitoring/v1alpha1/scrapeconfig_types.go

```diff
@@ -144,6 +144,8 @@ type ScrapeConfigSpec struct {
 	// Only valid in Prometheus versions 2.27.0 and newer.
 	// +optional
 	LabelValueLengthLimit *uint64 `json:"labelValueLengthLimit,omitempty"`
+	// MetricRelabelConfigs to apply to samples before ingestion.
+	MetricRelabelConfigs []*v1.RelabelConfig `json:"metricRelabelings,omitempty"`
```

Comment:
Small nit: we should include +optional annotation

Reply:
Very small nit but I'm not sure if we need a pointer here since default value for array in go lang is empty.
wdyt @ArthurSens and @simonpasquier ?